lib/types.d.ts

```diff
@@ -1,10 +1,4 @@
-import {IsEqual} from 'type-fest';
-
-/**
- * Represents a merged object between `TObject` and `TOtherObject`.
- * If both objects have a property with the same name, the type from `TOtherObject` is used.
- */
-type MergeBy<TObject, TOtherObject> = Omit<TObject, keyof TOtherObject> & TOtherObject;
+import {IsEqual, Merge} from 'type-fest';
```

Comment:
VSCode give me an error "Module '"type-fest"' has no exported member 'IsEqual'."

Reply:
Here's what their example shows:
```ts
import type {IsEqual} from 'type-fest';
```

Reply:
Hmmm strange, I'm not having this error @neil-marcellini. Will check it tomorrow with @blazejkustra

Reply:
@neil-marcellini Did you install dependencies & restarted VScode? This error showed up when you were in which repo? Expensify/App or react-native-onyx?

Reply:
I'll check again. I didn't install the dependencies so that was probably it.

Reply:
Yep that fixed it 😅